src/adapter-manager.js

```diff
@@ -397,15 +396,8 @@ class AdapterManager extends EventEmitter {
       newSettings.moziot.config = Object.assign(manifest.moziot.config || {},
                                                 savedSettings.moziot.config);
     } else {
-      // Default the Zigbee and Z-Wave adapters to enabled for now.
-      const defaults = [
-        'moziot-adapter-zigbee',
-        'moziot-adapter-zwave',
-      ];
-
-      if ((defaults.includes(manifest.name) && !testMode) ||
-          (manifest.moziot._test && testMode)) {
-        newSettings.moziot.enabled = true;
+      if (manifest.moziot.hasOwnProperty('enabled')) {
+        newSettings.moziot.enabled = manifest.moziot.enabled;
```

Comment:
This branch is unnecessary, since `newSettings` is just a copy of `manifest` at this point.